src/libmld/ldfsymorder.c

```diff
@@ -1,281 +1,93 @@
-__asm__(R""(
-.macro glabel label
-    .global \label
-    .balign 4
-    \label:
-.endm
+#include "libmld.h"
+
+int ldfsymorder(HDRR* hdr, int offset) {
```

Comment:
For the rest of the functions as well
```suggestion
/*
0048D0E0 st_readst
*/
int ldfsymorder(HDRR* hdr, int offset) {
```